src/testcafe-runner.js

```diff
@@ -42,19 +46,53 @@ async function run (runCfgPath, suiteName) {
         'list'
       ]);
 
-    if (!process.env.SAUCE_VM || process.env.SAUCE_VIDEO_RECORD) {
+    if (suite.tsConfigPath) {
+      runnerInstance.tsConfigPath(path.join(projectPath, suite.tsConfigPath));
+    }
+
+    if (suite.clientScripts) {
+      let clientScriptsPaths = Array.isArray(suite.clientScripts) ? suite.clientScripts : [suite.clientScripts];
+      clientScriptsPaths = clientScriptsPaths.map((clientScriptPath) => path.join(projectPath, clientScriptPath));
+      runnerInstance.clientScriptPath(clientScriptsPaths);
+    }
+
+    // Record a video if it's not a VM or if SAUCE_VIDEO_RECORD is set
+    const shouldRecordVideo = !suite.disableVideo && (!process.env.SAUCE_VM || process.env.SAUCE_VIDEO_RECORD);
+    if (shouldRecordVideo) {
       runnerInstance.video(assetsPath, {
         singleFile: true,
         failedOnly: false,
         pathPattern: 'video.mp4'
       });
     }
 
+    // Screenshots
+    if (suite.screenshots) {
+      runnerInstance.screenshots({
+        ...suite.screenshots,
+        path: assetsPath,
+        // Set screenshot pattern as fixture name, test name and screenshot #
+        // This format prevents nested screenshots and shows only the info that
+        // a Sauce session needs
+        pathPattern: '${FIXTURE}__${TEST}__screenshot-${FILE_INDEX}',
+      });
+    }
+
     results = await runnerInstance.run({
-      disablePageCaching: process.env.DISABLE_PAGE_CACHING || true,
-      disableScreenshot: process.env.DISABLE_SCREENSHOT || true,
-      quarantineMode: process.env.QUARANTINE_MODE || false,
-      debugMode: process.env.DEBUG_MODE || false
+      skipJsErrors: suite.skipJsErrors,
+      quarantineMode: suite.quarantineMode,
+      skipUncaughtErrors: suite.skipUncaughtErrors,
+      selectorTimeout: suite.selectorTimeout,
```

Comment:
@dpgraham are we setting defaults for these values?

Reply:
@vrunoa we don't need to. The `run` method has defaults. So if we pass "undefined", it will use the built-in default.